readthedocs/projects/version_handling.py

```diff
@@ -127,7 +127,7 @@ def comparable_version(version_string):
         elif version_string == STABLE_VERBOSE_NAME:
             comparable = Version('9999.0')
         else:
-            comparable = Version('999.0')
+            comparable = Version('0.01')
```

Comment:
This change may warrant an update to the docstring above, as we are assuming it's less than all other versions.